cli/src/command.rs

```diff
@@ -415,6 +477,14 @@ pub fn run() -> Result<()> {
 			#[cfg(not(feature = "polkadot-native"))]
 			panic!("No runtime feature (polkadot, kusama, westend, rococo) is enabled")
 		},
+		#[cfg(not(debug_assertions))]
+		Some(Subcommand::HostPerfCheck) => {
+			let mut builder = sc_cli::LoggerBuilder::new("");
```

Comment:
Could you elaborate on `""` ?

Reply:
It defaults to "info" as far as I understand.
Do you propose to make it explicit `"info"`?

Reply:
Explicit is better than implicit.